### ISSUE TITLE
parsl 2024.12.09 breaks `ComputeManager`

### ISSUE THREAD

**Issue by mattwthompson** [MolSSI/QCFractal#864]
**Describe the bug**

**To Reproduce**

```shell
$ micromamba create --name qcfractal qcfractalcompute -c conda-forge -yq && micromamba activate qcfractal
$ python -c "from qcfractalcompute.compute_manager import ComputeManager"
Traceback (most recent call last):
  File "<string>", line 1, in <module>
    from qcfractalcompute.compute_manager import ComputeManager
  File "/Users/mattthompson/micromamba/envs/qcfractal/lib/python3.13/site-packages/qcfractalcompute/__init__.py", line 9, in <module>
    from .compute_manager import ComputeManager
  File "/Users/mattthompson/micromamba/envs/qcfractal/lib/python3.13/site-packages/qcfractalcompute/compute_manager.py", line 37, in <module>
    from .executors import build_executor
  File "/Users/mattthompson/micromamba/envs/qcfractal/lib/python3.13/site-packages/qcfractalcompute/executors.py", line 7, in <module>
    from parsl.channels import LocalChannel
ModuleNotFoundError: No module named 'parsl.channels'
```

This error does not show up after downgrading Parsl:

```shell
$ micromamba install "parsl <2024.12.09" -c conda-forge -yq
$ python -c "from qcfractalcompute.compute_manager import ComputeManager"
$
```

**Expected behavior**

Not this error

**Additional context**

https://github.com/Parsl/parsl/pull/3716

**Comment by bennybp** [comment]
Already fixed in PR #863 :)

There will be a new release either later this week or early next week

**Comment by mattwthompson** [comment]
Nice! ... I need to find better search terms
